firebase-firestore/src/main/java/com/google/firebase/firestore/MemoryLruGcSettings.java

```diff
@@ -30,20 +37,42 @@ public MemoryLruGcSettings build() {
       return new MemoryLruGcSettings(sizeBytes);
     }
 
-    public void setSizeBytes(long size) {
+    /**
+     * Sets an approximate cache size threshold for the memory cache. If the cache grows beyond this
+     * size, Firestore SDK will start removing data that hasn't been recently used. The size is not
+     * a guarantee that the cache will stay below that size, only that if the cache exceeds the
+     * given size, cleanup will be attempted.
+     *
+     * <p>A default size of 100MB is used if unset. The minimum value to set is 1 MB.
```

Comment:
nit: "1 MB" -> "N,NNN,NNN" (you can omit the units since it is documented to be in bytes)